src/main/scala/it/pagopa/interop/tenantmanagement/model/persistence/serializer/v1/protobufUtils.scala

```diff
@@ -13,6 +13,8 @@ import it.pagopa.interop.tenantmanagement.model.tenant.PersistentTenantMailKind.
 
 object protobufUtils {
 
+  private val emptyName: String = ""
+
```

Comment:
There's not much value in having it defined here as a variable, can you inline it?